src/concrete/ml/quantization/quantized_module.py

```diff
@@ -598,9 +599,15 @@ def compile(
                 error to a default value.
             verbose (bool): Indicate if compilation information should be printed
                 during compilation. Default to False.
+            inputs_encryption_status (Optional[Sequence[str]]): encryption status ('clear',
+                'encrypted') for each input.
 
         Returns:
             Circuit: The compiled Circuit.
+
+        Raises:
+            ValueError: if inputs_encryption_status does not match with the
```

Comment:
it's a bit more than that right ?

Reply:
Well it's if the configuration is wrong, how would you formulate it?

Reply:
I would usually very quickly describe what are bad configurations